### PR TITLE
/bank: add Founder's Toolbox, and general updates & improvements to the page

### PR DESCRIPTION
**Pull request by exu3**
This PR adds nonprofits & the Founder's Toolbox to the bank page

### CODE REVIEW

File: components/bank/Everything.js
Comment (cfanoulis) [inline components/bank/Everything.js]: Can we include Google pay here?
```suggestion
              'Virtual debit cards (compatible with Apple Pay & Google Pay)': '',
```

**Review by cfanoulis — COMMENTED** [review]
following wording changes, also, can the commented code just be removed? it will exist in the git commit history anyways

**Comment by exu3** [comment]
Thanks for the review @cfanoulis! I commented it out because I'm not 100% sure about removing it yet. I know that there's also a lot of code commented out in other places too but I plan to go through & remove that before the PR gets merged. Just opened this PR to get a preview deploy to send to some people

**Comment by cfanoulis** [comment]
ah cool just saw it pop up in #bot-spam and thought i should take a look

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/bank**](https://v3-nvavfoodl.hackclub.dev/bank) | [**/donate**](https://v3-nvavfoodl.hackclub.dev/donate) |
|:-:|:-:|
|<a href="https://v3-nvavfoodl.hackclub.dev/bank">Screenshot of /bank</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/2Kt7Q1Pn25NNcsU6-MvaIiLhFL6hYD0Qr5llHSv0sQKsZ3psYF2VKBhzah5LsMk5ToO146vJTAARk1M01VydnQ.png">(view full size)</a> |<a href="https://v3-nvavfoodl.hackclub.dev/donate">Screenshot of /donate</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/NsEVoKX78ymtXx5exwi57sR3nsAxctu2ZhpCSZX3LqEA1zKUQN2_jeIZ7dtdCqbMCymbQmx1H1TzYovqbiHWXA.png">(view full size)</a> |



| [**/**](https://v3-nvavfoodl.hackclub.dev/) | [**/philosophy**](https://v3-nvavfoodl.hackclub.dev/philosophy) |
|:-:|:-:|
|<a href="https://v3-nvavfoodl.hackclub.dev/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/ttPQ9-LO72DjSINnDonI45iVQ7BKqrMOZzg2Ehzt4L1lJfU90jse5yOLboxuW0QvKVLA430NycSrYTBZGC9MWg.png">(view full size)</a> |<a href="https://v3-nvavfoodl.hackclub.dev/philosophy">Screenshot of /philosophy</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/42IDJPw7pEdEBFk9e00FPYIuRa4ng5yet3GPlTdbBT7MMAIWrsAhc9Ic3C199OOPA_8jYz4HzYhpEK9g66fftQ.png">(view full size)</a> |



| [**/team**](https://v3-nvavfoodl.hackclub.dev/team) |
|:-:|
|<a href="https://v3-nvavfoodl.hackclub.dev/team">Screenshot of /team</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/i2FeXKv2vZZDwe1eaJrtSCdGTKvf6-sUAlL7arJ7aJAOlL52KvwByXx_ig56SUna_2S7pZXuB1JBxKIdSbBUAA.png">(view full size)</a> |







Commit 5609afc4cba1b1a342ffaca8d55b5e48ab5e64ad (https://v3-nvavfoodl.hackclub.dev).

**Review by sampoder — COMMENTED** [review]
I had a couple of thoughts on this PR (it looks pretty cool but I had some thoughts on the integration), tried to make a concise video but it kinda failed... https://youtu.be/7MmMIh-YRvg. Any way there are some of my thoughts on it

**Comment by exu3** [comment]
hey @sampoder! super appreciate the detailed and straightforward feedback - I thought it was very helpful, and I agree with a lot the things you pointed out. I made a few design changes if you'd like to take another look; it still feels a little bit off to me (can't put a word to it at the moment but maybe i'll find something later). thanks so much for the screen recording!

**Comment by sampoder** [comment]
Hmm, maybe we don't need "and a suite of free perks..."

**Review by zachlatta — APPROVED** [review]
Spectacular work @eilla1. Great, great work on this. I think it is looking so good, and pushed a few minor tweaks myself.

I think there are two final things that we need to do, then we can merge!

- I noticed that some parts of the page don't scale super well on mobile. Can you do a deep pass on that, to make sure it displays great on all screen sizes (and, in particular, on phones)?

- I'd like @maxwofford to do a final code quality review with you. Maybe you two could set up a Zoom call and pair program on it. I am honestly not very familiar with React / modern web best practices, so I'm not sure if this will be super minor or take an hour or so.

Fantastic work. Let's get this thing merged and live! And make sure you write up an awesome post in `#ship` when we're live.

**Comment by maxwofford** [comment]
@eilla1 will message you on Slack about setting up a call!

**Review by sampoder — APPROVED** [review]
clearing my review request, Ella this is awesome!

**Review by cfanoulis — APPROVED** [review]
Clearing my review as well, LGTM!

**Comment by maxwofford** [comment]
Just checked over the code with @eilla1 – cool with this getting merged!

**Comment by sampoder** [comment]
let's do it

**Pull request merged**
